### PR TITLE
Fix link to Ruby-Doc.org

### PR DESCRIPTION
**Pull request by laurenskrom**

### CODE REVIEW

File: ruby_programming/basic_ruby/basic_data_types.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ puts 'Hello, #{name}' #=> "Hello, #{name}"
 #### Common String Methods
 There are many useful string methods that are built into Ruby. You need to capitalize a word? No problem! Reverse a string? Easy peasy. Extract the binary subatomic algorithm from any regex grep? We don't know, but since this is Ruby, let's go with *YES*.
 
-Just remember, strings have loads of methods provided to you for free, and you can find them all in the [Ruby docs](ruby-doc.org/core-2.6.0/String.html). If you're working with strings and need to do something, check the Ruby docs first and see if there's a method that does it for you.
+Just remember, strings have loads of methods provided to you for free, and you can find them all in the [Ruby docs](https://ruby-doc.org/core-2.6.0/String.html). If you're working with strings and need to do something, check the Ruby docs first and see if there's a method that does it for you.
```
Comment (I3uckwheat) [inline ruby_programming/basic_ruby/basic_data_types.md:160]: This link takes me to a 404

Comment (laurenskrom) [reply ruby_programming/basic_ruby/basic_data_types.md:160]: I think the original author was looking for this one: https://ruby-doc.org/core-2.6/String.html

Comment (laurenskrom) [reply ruby_programming/basic_ruby/basic_data_types.md:160]: But I don't know how to change it now, should i start a new PR?

Comment (I3uckwheat) [reply ruby_programming/basic_ruby/basic_data_types.md]: @laurenskrom You can edit this PR, just make changes and push to the same branch. Or use the Github interface

**Comment by I3uckwheat** [comment]
@laurenskrom Awesome, thanks!

**Pull request merged**